packages/General/General.0.7.0/opam

```diff
@@ -0,0 +1,25 @@
+opam-version: "2.0"
+synopsis: "Rich functionality for built-in and basic OCaml types"
+maintainer: "Vincent Jacques <vincent@vincent-jacques.net>"
+authors: "Vincent Jacques <vincent@vincent-jacques.net>"
+license: "MIT"
+homepage: "https://jacquev6.github.io/General/"
+doc: "https://jacquev6.github.io/General/"
+bug-reports: "http://github.com/jacquev6/General/issues/"
+depends: [
+  "ocaml" {>= "4.02.3"}
+  "dune" {build}
```

Comment:
```suggestion
  "dune"
```
dune should not be tagged as a build dependency anymore (see #14266)

Reply:
Thanks :)